docs/guides/hooks-list.rst

```diff
@@ -115,6 +115,12 @@ System hooks
 
 **add, river**
 
+**elgg.data, site**
```

Comment:
I suppose you want them to be consistent with JS naming convention, but that also bring discrepancy to hook names we have server-side